setup.cfg

```diff
@@ -60,3 +60,8 @@ silx.resources =
     opencl/codec/*.cl
     gui/colormaps/*.npy
 silx.examples = *.png
+
+[flake8]
+max-line-length = 88
+extend-ignore = E203, W503
+exclude = .venv,build,dist
```

Comment:
There's already a `.flake8` file to configure flake8 so this should not be needed.
Could you revert this?

On top of that, there is also an opened Pull Request to change the build system from setuptools to meson, so the `setup.cfg` file will be removed.